CondCore/CondHDF5ESSource/README.md

```diff
@@ -0,0 +1,49 @@
+# Overview of CondCore/CondHDF5ESSource
+
+## CondHDF5ESSource
+
+CondHDF5ESSource is an EventSetup source that can read conditions data from an HDF5 file. The full configuration options for the module can be obtained by doing the standard
+
+```
+> edmPluginHelp -p CondHDF5ESSource
+```
+
+In general, one will specify the file to read and the global tag stored within that file to use. E.g.
+
+```python
+
+globalTag = cms.ESSource("CondHDF5ESSource", filename = cms.string("JobConditions.h5cond"), globalTag = cms.string("SomeTag") ) 
```

Comment:
I guess, in the scope of easing the life of HPC sites with limited external connection, we should also think about making this option available/configurable at the level of cmsDriver commands (for full productions etc...)?

Reply:
Ok sounds good to me, indeed this is what we also discussed at the AlCaDB meeting ~2 weeks ago.